In [2]:
import pandas as pd
import matplotlib.pyplot as plt

In [3]:
df = pd.read_csv('../data/output/separations.csv')
df

,pair,tertiary,separation,class
0,HH270VLA1,False,99.4,C0
1,HOPS-12,True,1853.5,C0
2,HOPS-12,False,80.6,C0
3,HOPS-168,False,73.7,C0
4,HOPS-173,False,103.0,C0
5,HOPS-182,False,381.5,C0
6,HOPS-193,False,100.0,C1
7,HOPS-203,False,49.7,C0
8,HOPS-203,True,1085.5,C0
9,HOPS-213,False,114.4,FS


In [7]:
df[df['tertiary'] == False].describe()

,separation
count,36.000000
mean,154.886111
std,136.258677
min,24.100000
25%,68.950000
50%,99.600000
75%,195.850000
max,572.500000


Among the 36 fields with measured outflow PAs, the maximum binary separation is 572.5 AU, the minimum is 24.1 AU, and the median is 99.6 AU.